packages/ffe-file-upload/less/file-upload.less

```diff
@@ -0,0 +1,304 @@
+@ffe-blue-royal-escape: escape('@{ffe-blue-royal}');
+@ffe-blue-royal-fill: @ffe-blue-royal-escape;
+@ffe-blue-azure-escape: escape('@{ffe-blue-azure}');
+@ffe-blue-azure-fill: @ffe-blue-azure-escape;
+
+.ffe-file-upload {
+    margin-top: 45px;
+    display: block;
+
+    @media (min-width: @breakpoint-md) {
+        display: -webkit-box;
+        display: -ms-flexbox;
+        display: flex;
+    }
+
+    input[type='file'] {
+        display: none;
+    }
+
+    &__info-section {
+        max-width: 400px;
+
+        @media (min-width: @breakpoint-md) {
+            margin-right: 20px;
+            max-width: 100%;
+            -webkit-box-flex: 0;
+            -ms-flex: 0px;
+            flex: 0px;
+        }
+
+        &__title {
```

Comment:
This goes for much of the styling structure - Multi level nesting of elements should be avoided, and the class names should not reflect the dom structure. In stead, each element in a block should be independent (but they can still be nested in the markup). See [Why not create elements of elements?](https://en.bem.info/methodology/faq/#why-not-create-elements-of-elements-block__elem1__elem2) and [BEM/Nesting](https://en.bem.info/methodology/quick-start/#nesting-1)

Reply:
Will rewrite this when possible, didn't have much time to spare before :)